### PR TITLE
vv tests with new API closes #83

### LINKED ISSUE #83: refactor vv code to use the new api (subset of docs)
refactor the vv so it trains the tf-idf matrix from >1000 docs and gets results by row only. That can be done after #73

### PR DESCRIPTION
**Pull request by thanasions**

### CODE REVIEW

File: tests/algorithms/test_tfidf_vv.py [hunk 1]
```diff
@@ -53,8 +54,8 @@ def setUpClass(cls):
         cls.n_tests = 0
     
     def find_idx(self, textin):
-        for i, text in enumerate(self.df):
-            if text == textin:
+        for i, text in enumerate(self.test_dataframe['abstract']):
+            if Lev.distance(text, textin)/len(text) <0.1:
```
Comment (IanGrimstead) [inline tests/algorithms/test_tfidf_vv.py]: Why the change? Used to match 1:1

Comment (thanasions) [reply tests/algorithms/test_tfidf_vv.py]: used to, not any more :) i debugged and checked on the calculator. Not sure why they were not the same tbh. Levenstein distance of 35 for the first patent, meant they were 35 chars apart. I thought to introduce Lev distances as we may need them elsewhere too..
